### PR TITLE
SDK-2024: Update doc

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-2028

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,79 +5,6 @@
...
-2. For example, add `NotUnique:Constant` check to ignore list:
-
-```yaml
-evaluator:
-  rules:
-    ignore:
-      - NotUnique:Constant
-```
-
-
```
Comment (DmytroKlymanSpryker) [inline README.md:80]: Should we have here somewhere the link on new file [/evaluator.md](https://github.com/spryker-sdk/upgrader/pull/128/files#diff-9df6d20ecace5c2636d8e9d9a3785572b2b6e00d3ec25519c41a3aa326206079) if we keep it in repo?

Comment (VladislavStrelchenko) [reply README.md:80]: Added link to evaluator.md

File: README.md [hunk 2]
```diff
@@ -244,7 +171,11 @@ bin/console upgradability:php:upgrade
 export INTEGRATOR_ENABLED=true
 ```
 
-​
+## Evaluator
+
+[Evaluator documentation](https://github.com/spryker-sdk/upgrader/blob/master/docs/evaluator.md)
```
Comment (DmytroKlymanSpryker) [inline README.md:176]: For links on the docs inside the same repo use the relative paths: 

[Evaluator documentation](docs/evaluator.md).

**Comment by DmytroKlymanSpryker** [comment]
Should be released after: https://github.com/spryker/spryker-docs/pull/1675

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**